AIDevGallery.Utils/ModelUrl.cs

```diff
@@ -252,6 +252,10 @@ public static string GetFullUrl(string url)
         {
             return url;
         }
+        else if (url.StartsWith("local", StringComparison.InvariantCultureIgnoreCase))
```

Comment:
>StringComparison.InvariantCultureIgnoreCase [](http://example.com/codeflow?start=41&length=43)

nit: I think you need to use `OrdinalIgnoreCase` better. It is faster and avoid running into linguistic cases in the comparison. I am seeing InvariantCulture used in many places which doing the operation as ordinal comparison make more sense.

Reply:
Thank you @tarekgh - I updated all instances